WordPress/src/main/res/xml-v31/stats_week_views_widget_info.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:configure="org.wordpress.android.ui.stats.refresh.lists.widget.weeks.StatsWeekWidgetConfigureActivity"
+    android:initialKeyguardLayout="@layout/stats_widget_list_light"
+    android:initialLayout="@layout/stats_widget_list_light"
+    android:minHeight="110dp"
+    android:minResizeWidth="150dp"
+    android:minWidth="250dp"
+    android:previewLayout="@layout/stats_week_views_widget_preview"
+    android:resizeMode="horizontal"
+    android:updatePeriodMillis="600000"
+    android:widgetCategory="home_screen"
+    tools:ignore="UnusedResources">
+</appwidget-provider>
```

Comment:
np: Since `stats_week_views_widget_info.xml` is used only in `jetpack/AndroidManifest.xml`, can we move `stats_week_views_widget_info.xml` to a Jetpack directory for adding this file only to the Jetpack apk.